MobileWallet/Assets.xcassets/Images/GemAmount.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "GemAmount.png",
```

Comment:
Can use the current `Gem` image asset and just set the tint. Like what's happening in the TX detail view.